flake.nix

```diff
@@ -54,101 +56,51 @@
     treefmt-nix.url = "github:numtide/treefmt-nix";
   };
 
-  outputs =
-    inputs@{ flake-parts, self, ... }:
-    flake-parts.lib.mkFlake { inherit inputs; } {
-      systems = import inputs.systems;
-
-      imports = [
-        ./modules
-        inputs.lite-config.flakeModule
-        inputs.treefmt-nix.flakeModule
-      ];
-
-      lite-config =
-        { lib, ... }:
-        {
-          nixpkgs = {
-            config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [ "terraform" ];
-            overlays = [
-              (final: prev: {
-                hydra = (prev.hydra.override { nix = final.nixVersions.nix_2_24; }).overrideAttrs (o: {
-                  version = inputs.hydra.shortRev;
-                  src = inputs.hydra;
-                  buildInputs = o.buildInputs ++ [ final.perlPackages.DBIxClassHelpers ];
-                });
-              })
-            ];
-          };
+  # overlays = [
+  #   (final: prev: {
+  #     hydra = (prev.hydra.override { nix = final.nixVersions.nix_2_24; }).overrideAttrs (o: {
+  #       version = inputs.hydra.shortRev;
+  #       src = inputs.hydra;
+  #       buildInputs = o.buildInputs ++ [ final.perlPackages.DBIxClassHelpers ];
+  #     });
```

Comment:
This could be moved to `packages/hydra/default.nix`, and then pulled in the module with `services.hydra.package = perSystem.self.hydra`;

Reply:
With lite-config this overlay is applied to `pkgs`so we don't need to do any additional wiring, e.g. `.#nixosConfigurations.build03` and `.#checks.x86_64-linux.nixosTests-hydra` both just work. It is also exposed directly in `.#packages.${system}.hydra`.